### DISCUSSION TITLE
Skip Link Questions

### DISCUSSION THREAD

**Discussion by 0618** [jupyter/accessibility#38] [Q&A]
For issue [#9688](https://github.com/jupyterlab/jupyterlab/issues/9688) Add Skip Link to All Regions and Add Region Aria-Role to Panels, I got some questions.

I might missed some docs, but based on my current understanding, we don't really have many items need to be skipped, do we?

If we do, can anyone help me understanding our **current key navigation** and our **expected skip link behavior**?

When investigating, I assumed that we need to skip the tab key navigation on the page. The skip link can be as simple as 1 link to the active tab content. Or having several links to the main-menu, left-bar, and active tab.

Here are the questions I got:
![skip link](https://user-images.githubusercontent.com/11983489/112259191-12155a80-8c25-11eb-95a1-aa5bbbbb1e35.png)

**Comment by trallard** [comment]
@gabalafou  or @isabela-pf  might have some insight here - they have been looking at these types of interactions recently
